packages/definitions-parser/test/git.test.ts

```diff
@@ -53,17 +48,10 @@ You should only delete files that are a part of removed packages.`,
   async extraneousFile() {
     expect(
       await getNotNeededPackages(allPackages, [
-        { status: "A", file: "oooooooooooops.txt" },
-        { status: "M", file: "notNeededPackages.json" },
-        { status: "D", file: "types/jest/index.d.ts" },
-        { status: "D", file: "types/jest/jest-tests.d.ts" },
+        ...deleteJestDiffs,
+        { status: "A", file: "types/oops/oooooooooooops.txt" },
       ])
-    ).toEqual({
-      errors: [
-        `Unexpected file added: oooooooooooops.txt
```

Comment:
I can't figure out why getNotNeededPackages was complaining about *added* files in the first place. Maybe it was left over from when gitDiff also processed added files?

Reply:
It still does here: https://github.com/microsoft/DefinitelyTyped-tools/pull/787/files#diff-851343706958307266e65b65ada1241e0e87962c124efbd7a501d219d724a534R67

Needed to make sure we attempt to run tests on dirs that _look like_ they're supposed to be packages. Otherwise CI passes for old style without package.json (the original change was fixing that).

Reply:
But, the test doesn't seem to show that we are adding a package here for `oops`, so something may still be wrong...

Reply:
It may not matter since almost all additions are going to be (1) a single file to an existing types directory (2) multiple files attempting to add a new types directory. Mostly what we want to catch are incomplete cases of (2), and I don't *think* that's happening here in the first place.

Reply:
Actually, no, I can't find any other tests for incomplete packages. I don't even know anymore.